drag_and_drop_v2/utils.py

```diff
@@ -5,3 +5,24 @@
 # Make '_' a no-op so we can scrape strings
 def _(text):
     return text
+
+
+def ngettext(text_singular, text_plural, n):
+    if n == 1:
+        return text_singular
+    else:
+        return text_plural
+
+
+class FeedbackMessages(object):
+    FINAL_ATTEMPT_TPL = _('Final attempt was used, highest score is {score}')
+    MISPLACED_ITEMS_RETURNED = _('Misplaced items were returned to item bank.')
+
+    CORRECTLY_PLACED_SINGULAR_TPL = _('Correctly placed {correct_count} item.')
+    CORRECTLY_PLACED_PLURAL_TPL = _('Correctly placed {correct_count} items.')
```

Comment:
@e-kolpakov I tried and it didn't work. Check OC-1802 for some more info. Getting translations to work is completely out of scope of this PR, I think we should just mark the strings for extraction correctly, so that when XBlock i18n cli tools are developed and state of XBlock i18n support improved, we will be ready to go.